brave/browser-laptop
# Upgrade webpack to 3.8

For me it speeds up `CHANNEL=dev npm run build-package`:

webpack | 1st build | Subsequent builds
-|-|-
3.4.1 | 85s | 65s
3.8.1 | 65s | 35s

Related to #11476

Submitter Checklist:


Test Plan:
- Try to build a package and run it, on MacOS, Linux and Windows.
- Try to develop (`npm run watch` and `npm run start`) on MacOS, Linux and Windows.
- Check automated tests still run.

Reviewer Checklist:

Tests
